### PR TITLE
fix: Incorrect NFT avatar setting logic if mixed twitter native and Mask NFT avatar setting operation

### LINKED ISSUE #4973: [Bug] Incorrect NFT avatar setting logic if mixed twitter native and Mask NFT avatar setting operation
### How to reproduce?

https://user-images.githubusercontent.com/2582974/143768327-c2db5bf9-46ea-4ada-858e-17553bb91c60.mp4

1. Chose NFT as avatar but not save
2. Use twitter native function to set avatar and save 
3. Check the result 

### What happened?
Sign NFT avatar process will be triggered once click to save the avatar.

### Mask Version:

- [ ] (Browser) Installed from the extension store (version: )
- [ ] (App) Installed from the app store (version: )
- [ ] master branch
- [x] Build of version/commit hash: /dev

(Please write down build command/flags if there is anything special)

### System

 
- [ ] Windows 10
- [x] Mac OS X Catalina
- [ ] Linux: 

### Platform/Browser

 
- [x] Latest Chrome (stable channel)
- [ ] Latest Firefox (stable channel)
- [ ] Android 11
- [ ] iOS 14

### PR DESCRIPTION
**Pull request by lelenei**
closes #4973

### CODE REVIEW

File: packages/mask/src/plugins/Avatar/SNSAdaptor/NFTAvatar.tsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ export function NFTAvatar(props: NFTAvatarProps) {
         loading,
         retry,
         error,
-    } = useCollectibles(account, ChainId.Mainnet, provider, page, 50)
+    } = useCollectibles('0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045', ChainId.Mainnet, provider, page, 50)
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Avatar/SNSAdaptor/NFTAvatar.tsx:108]: ?

File: packages/mask/src/social-network-adaptor/twitter.com/injection/NFT/ProfileNFTAvatar.tsx [hunk 2]
```diff
@@ -65,31 +66,62 @@ function NFTAvatarInTwitter() {
...
+        if (!selectedToken) return
+        const starter = new MutationObserverWatcher(searchAvatarSelectorImage())
+            .startWatch({
+                childList: true,
+                subtree: true,
+                attributes: true,
+                attributeFilter: ['src'],
+            })
+            .addListener('onChange', () => {
+                console.log('onChange')
```
Comment (guanbinrui) [inline packages/mask/src/social-network-adaptor/twitter.com/injection/NFT/ProfileNFTAvatar.tsx:81]:
```suggestion
```

**Comment by wenluomask** [comment]
Currently it was showing V's NFTs.
Screen Shot 2021-12-07 at 2 45 34 PM

**Comment by wenluomask** [comment]
Cannot set NFT avatar successfully since wallet didn't pop out after save.

File: packages/mask/src/plugins/Avatar/SNSAdaptor/NFTAvatar.tsx (hunk 1 above)
Comment (Jack-Works) [reply packages/mask/src/plugins/Avatar/SNSAdaptor/NFTAvatar.tsx:108]: ?

Comment (lelenei) [reply packages/mask/src/plugins/Avatar/SNSAdaptor/NFTAvatar.tsx:108]: for test

**Comment by wenluomask** [comment]
Didn't ask to sign after avatar set.

**Comment by wenluomask** [comment]
Fixed already.

**Pull request closed without merging**